src/tests/ftest/cart/test_group_np_cli.c

```diff
@@ -30,11 +30,55 @@
 #include <assert.h>
 #include <getopt.h>
 #include <semaphore.h>
+#include <ctype.h>
 
 #include "tests_common.h"
 #include "test_group_rpc.h"
 #include "test_group_np_common.h"
 
+
+static void
+send_rpc_shutdown(crt_endpoint_t server_ep, crt_rpc_t *rpc_req)
+{
+	int rc = crt_req_create(test_g.t_crt_ctx[0], &server_ep,
+					CRT_PROTO_OPC(TEST_GROUP_BASE,
+						      TEST_GROUP_VER, 1), &rpc_req);
```

Comment:
(style)  line over 80 characters

Reply:
[Vim `ruler`](http://vimdoc.sourceforge.net/htmldoc/options.html#'ruler') says this line is 48 chars long, and a vim regex doesn't find any code lines (outside of the copyright notice at the top) exceeding 70 chars (`/\v^.{70,}`) in `test_group_np_cli.c`.